datasets/wikicorpus/wikicorpus.py

```diff
@@ -0,0 +1,179 @@
+}
+"""
+
+_DESCRIPTION = """\
+The Wikicorpus is a trilingual corpus (Catalan, Spanish, English) that contains large portions of the Wikipedia (based on a 2006 dump) and has been automatically enriched with linguistic information. In its present version, it contains over 750 million words.
+"""
+
+_HOMEPAGE = "https://www.cs.upc.edu/~nlp/wikicorpus/"
+
+_LICENSE = "GNU Free Documentation License"
+
+_URLs = "https://www.cs.upc.edu/~nlp/wikicorpus/{form}.{language}.tgz"
+
+_LANGUAGES = ["ca", "es", "en"]
+_FORMS = ["raw", "tagged"]
+
+METADATA_PATTERN = re.compile(r'.+id="(?P<id>[^"]+)".+title="(?P<title>[^"]+)".+')
+
+
+class WikicorpusConfig(datasets.BuilderConfig):
+    """ BuilderConfig for Wikicorpus."""
+
+    def __init__(self, form=None, language=None, **kwargs):
+        """
+        Args:
+            form: form of the dataset.
+            language: language of the dataset.
+            **kwargs: keyword arguments forwarded to super.
+        """
+        super().__init__(
+            name=f"{form}_{language}",
+            description=f"Wikicorpus dataset in {form} form and {language} language.",
+            **kwargs,
+        )
+        self.form = form
+        self.language = language
+
+
+class Wikicorpus(datasets.GeneratorBasedBuilder):
+    """Wikicorpus dataset."""
+
+    VERSION = datasets.Version("1.0.0")
+    BUILDER_CONFIG_CLASS = WikicorpusConfig
+    BUILDER_CONFIGS = [WikicorpusConfig(form=form, language=language) for form in _FORMS for language in _LANGUAGES]
+
+    def _info(self):
+        if self.config.form == "raw":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "title": datasets.Value("string"),
+                    "text": datasets.Value("string"),
+                }
+            )
+        elif self.config.form == "tagged":
+            features = datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "title": datasets.Value("string"),
+                    "sentence": datasets.Sequence(datasets.Value("string")),
```

Comment:
could you rename this column `tokens` ?

Reply:
let's keep it this way actually with the raw format of the dataset as for the POS tags